packages/terra-hookshot/docs/README.md

```diff
@@ -0,0 +1,36 @@
+# Terra Hookshot
+
+The Terra Hookshot component positions content according to a targeted attachment, ensuring they stay connected.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-hookshot`
+  - `yarn add terra-hookshot`
+
+## Usage
+
+```jsx
```

Comment:
We might want to "beef" up this example a bit to include an example of `<Hookshot.Content >` to pass through and then include or remove the `this.getTargetRef`, `this.handleOnPositions`, etc references that might be confusing here.

Reply:
Site page updated.